recipe/meta.yaml

```
@@ -89,8 +90,8 @@ outputs:
         # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
         # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
         - export NVCC="$(which nvcc)"                                                # [linux]
+        - export CUPY_NUM_BUILD_JOBS=1                                               # [(aarch64 or ppc64le)]
```

Seeing some issues with CI timing out on other archs. Similar issues were seen on CuPy's CI, which was fixed by decreasing threads used by `nvcc` ( https://github.com/cupy/cupy/pull/8461 ). Have started with reducing the number of build jobs, but we can also try limiting `nvcc` threads if CI is still struggling here

```suggestion
        - export CUPY_NUM_BUILD_JOBS=1                                               # [(aarch64 or ppc64le)]
        - export CUPY_NUM_NVCC_THREADS=1                                             # [(aarch64 or ppc64le)]
```

---

Looks like this resolve the issue. Otherwise CI gets oversubscribed similar to what we saw upstream:

* https://github.com/cupy/cupy/pull/8412
* https://github.com/cupy/cupy/pull/8461